kdc/connect.c

```diff
@@ -1199,15 +1187,22 @@ start_kdc(krb5_context context,
                 break;
             default:
 		for (i = 0; i < max_kdcs; i++) {
-		    if (pids[i] == (pid_t)-1) {
```

Comment:
ditto.

Reply:
Hence tolerate either 0 or -1 here, create as 0, and assign to -1 when clearing a slot, ensuring that we exercise both cases.  And log any unexpected behaviour if this still breaks.

Not religious on this point, but I think it may be slightly more resilient to bitrot.